docs/tutorials/setup.rst

```diff
@@ -84,10 +82,10 @@ the ``libraryDependencies`` list in your ``build.sbt``:
 .. code:: scala
 
     libraryDependencies ++= Seq(
-        "org.locationtech.geotrellis" %% "geotrellis-spark"  % "2.3.1",
-        "org.locationtech.geotrellis" %% "geotrellis-s3"     % "2.3.1", // now we can use Amazon S3!
-        "org.apache.spark"            %% "spark-core"        % "2.4.1" % "provided",
-        "org.scalatest"               %% "scalatest"         % "3.0.0" % "test"
+        "org.locationtech.geotrellis" %% "geotrellis-spark"    % "3.0.0",
+        "org.locationtech.geotrellis" %% "geotrellis-s3-spark" % "3.0.0", // now we can use the Amazon S3 store!
+        "org.apache.spark"            %% "spark-core"          % "2.4.1" % "provided",
+        "org.scalatest"               %% "scalatest"           % "3.0.0" % "test"
```

Comment:
Hm I noticed that these are outdated, I will update them.

Reply:
What is out of date here? I believe the geotrellis and spark dependencies are up to date as listed here.

Reply:
@CloudNiner 

```scala
"org.apache.spark"            %% "spark-core"  % "2.4.3" % Provided,
"org.scalatest"                    %% "scalatest"     % "3.0.8" % Test
```

Reply:
Whoops. I stand corrected. Updated.